src/Types.php

```diff
@@ -221,10 +221,23 @@ public static function isFixedString($str)
         return strpos($str, 'fixedstring(') === 0;
     }
 
+    public static function isSimpleAggregateFunction($str)
+    {
+        return strpos($str, 'simpleaggregatefunction(') === 0;
+    }
+
+    protected function getTypeFromSimpleAggregateFunction($type)
```

Comment:
```php
protected function getTypeFromSimpleAggregateFunction(&$type)
```
need to get a type of presentation to the user in the `unFormat` method to perform formatting,
` SimpleAggregateFunction(max, DateTime64)`  uint64 => DateTime64